docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -56,6 +56,36 @@ Different settings can be applied to a load balancer service in AWS using _annot
 
 The information for the annotations for AWS is taken from the comments on [aws.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go)
 
+# OpenStack
+This section describes all the possible configurations which can
+be used when using OpenStack with Kubernetes.
+
+## cloud.conf
+Kubernetes knows how to interact with OpenStack via the file cloud.conf. It is the file that will provide Kubernetes with credentials and location for the OpenStack auth endpoint.
+You can create a cloud.conf file by specifying the following details in it
+
+### Minimal configuration
+This is an example of a minimal configuration that touches the values that most often need to be set:
+```yaml
+[Global]
+username=user  
+password=pass  
+auth-url=https://<keystone_ip>/identity/v3  
+tenant-id=c869168a828847f39f7f06edd7305637  
+domain-id=2a73b8f597c04551a0fdc8e95544be8a
+
+[LoadBalancer]
+subnet-id=6937f8fa-858d-4bc9-a3a5-18d2c957166a  
+```
+#### Global
```

Comment:
Put a blank line before ```yaml. Otherwise the YAML snippet does not render correctly. You can see this in the [Netlify preview](https://deploy-preview-5481--kubernetes-io-master-staging.netlify.com/docs/concepts/cluster-administration/cloud-providers/).

Reply:
@steveperry-53 Done.Thanks for the link but I see it still does not render correctly.